schemas/vertices/chemical_reactions.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["_key"],
+  "description": "Chemical reactions",
+  "properties": {
+    "_key": {
+      "type": "string",
+      "example": "rxn02201",
+      "description": "ms id",
+      "pattern": "^rxn\\d+$"
+    },
+    "direction": {
+      "type": "string",
+      "enum": [">", "<", "="]
+    },
+    "name": {
+      "type": "string",
+      "description": "Chemical names",
+      "example": "trans-2-Methyl-5-isopropylhexa-2,5-dienal dehydrogenase_c0"
+    },
+    "gpr": {
+      "type": "string",
+      "example": "PGN_RS01070"
+    },
+    "enzyme": {
+      "type": "string",
+      "example": "2.7.3.7",
+      "pattern": "^\\d+\\.\\d+\\.\\d+\\.\\d+$"
+    },
+    "bbcwn": {
+      "type": "number",
+      "example": -108
+    },
+    "equation": {
+      "type": "string",
+      "description": "Reaction formula using compound IDs (eg. cd00443)",
+      "example": "(1) cpd00443[c0] + (1) cpd02920[c0] => (1) cpd00012[c0] + (1) cpd00067[c0] + (1) cpd00683[c0]"
+    },
+    "definition": {
+      "type": "string",
+      "description": "Reaction formula. Same as equation, but with compound IDs replaced with chemical names."
+    },
+    "bigg_id": {
+      "type": "string",
+      "example": "DHPS2"
+    },
+    "kegg_id": {
+      "type": "string",
+      "example": "R03067"
+    },
+    "kegg_pathways": {
+      "type": "string",
+      "example": "Folate biosynthesis"
+    },
+    "metacyc_pathways": {
+      "type": "string",
+      "example": "AMINE-DEG|Creatinine-Degradation|Degradation"
```

Comment:
I think we probably want to update the uploader to split these | delimited fields (ids and pathways) into arrays in a future version

Reply:
We can do this now, might as well